hail/python/hailtop/hailctl/__main__.py

```diff
@@ -81,7 +82,7 @@ def fmt_version(version):
 
 
 def print_version():
-    print(hailctl.version())
+    print(version())
 
```

Comment:
Can just use `print(version())` in place.

Reply:
Sorry, I'm not understanding your comment, the `print_version` method was already here and updating `hailctl.version` -> `version` seems like the right change?

Reply:
Ya my comment wasn't very clear, just meant that having a separate function `print_version` seemed a bit unnecessary as opposed to writing `print(version())` later, but I didn't catch that you were just changing the definition. We can just leave this too